### DISCUSSION TITLE
concentration-dependent effect model

### DISCUSSION THREAD

**Discussion by MaLuisaEti** [Open-Systems-Pharmacology/Forum#981] [Models]
Hello,

We are working in a PBPK model for a drug which causes renal vasodilatation and it has a renal elimination. We are trying to optimize our model in MoBi considering the effect of plasma concentration of the drug on its renal excretion. What strategy do you recommend to model this using an Emax model? 

Thank you in advance
Best regards

María Luisa

**Comment by StephanSchaller** [comment]
Hi @MaLuisaEti, the thing is, you can model this pretty much any way you like in MoBi. So the answer depends on how much you are prepared to customize.
A simple question would be to make renal blood flow and GFR concentration-dependent. You can use Emax or any other kind of transfer function...

Best,
Stephan
